modules/crm_tools/decoupled_auth_crm_tools.module

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Simple implementation of Decoupled User Authentication Tools for CRM.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\ContentEntityBase;
+
+/**
+ * Implements hook_entity_presave().
+ *
+ * Update status log fields.
+ */
+function decoupled_auth_crm_tools_entity_presave(EntityInterface $entity) {
```

Comment:
Can this not live in the field `\Drupal\Core\Field\FieldItemListInterface::preSave`?

Reply:
So I tried all of the fielditemlist hooks... Unless the field actually is set pre save, nothing on the fielditemlist gets triggered.